paddle/operators/math/sequence2batch.h

```diff
@@ -159,6 +164,13 @@ class Batch2LoDTensorFunctor {
   }
 };
 
+template <typename Place, typename T>
+struct RowwiseAdd {
+  void operator()(const platform::DeviceContext& context,
+                  const framework::Tensor& input, const framework::Tensor& bias,
+                  framework::Tensor* output);
+};
```

Comment:
Why is this at sequence2batch.h?

Reply:
At first, this functor is only used in lstm_op and gru_op. Now I move it to `paddle/operators/math/math_function`.